src/main/java/racingcar/Main.java

```diff
@@ -0,0 +1,14 @@
+package racingcar;
+
+import racingcar.view.InputView;
+
+public class Main {
+
+  public static void main(String[] args) {
+
+    RacingGame racingGame = new RacingGame();
+    racingGame.readyGame(InputView.inputCount("자동차 대수는 몇 대 인가요?"));
+    racingGame.run(InputView.inputCount("시도할 회수는 몇 회 인가요?"));
```

Comment:
실행 질문에 대한 문자열은 변수를 통한 메세지 전달이 아닌 InputView클래스에 선언해서 사용해보시는건 어떠실지 의견드립니다 : )
```suggestion
    racingGame.readyGame(InputView.inputCount());
    racingGame.run(InputView.inputCount());
```